malicious_category.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package msg;
+
+option go_package = "zvelo.io/msg";
+
+enum MaliciousCategory {
+  T_SKIP = 0;
+  T_BOTS_4 = 10293;
+  T_CANDC_4 = 10294;
+  T_COMPR_4 = 10295;
+  T_MALHOME_4 = 10296;
+  T_MAL_4 = 10297;
+  T_FRAUD_4 = 10298;
+  T_SPAM_4 = 10299;
+  T_SPYWARE_4 = 10300;
+  T_UNKNOWN = -1;
```

Comment:
T- "Type". Since these are the same name/categories which are used by MASCAT and exist in category.proto. I have created a story to remove it later from MASCAT since we dont know where it is used.